src/components/Theme/src/default-theme.js

```diff
@@ -15,6 +15,21 @@ export default function defaultTheme() {
 			elevation: '#ffffff',
 			overlay: 'rgba(0, 0, 0, 0.3)',
 			primary: '#000000',
+			critical: {
+				fill: '#cd2026',
+				text: '#a82826',
+				subtle: '#f6eceb',
```

Comment:
It's not clear in the code or documentation how fill or subtle are used. We're directly referencing text, but how do the others work?